src/baldr/accessrestriction.cc

```diff
@@ -98,6 +99,47 @@ const json::MapPtr AccessRestriction::json() const {
   return map;
 }
 
+void AccessRestriction::rapidjson(rapidjson::writer_wrapper_t& writer) const {
```

Comment:
there is no point to keeping the above function and this function. also i think it should still be called `json` and not `rapidjson`